internal/venice-client-common/src/main/java/com/linkedin/venice/compression/CompressorFactory.java

```diff
@@ -12,8 +12,8 @@
 
 public class CompressorFactory implements Closeable, AutoCloseable {
   private static final Logger LOGGER = LogManager.getLogger(CompressorFactory.class);
-  private static final VeniceCompressor NO_OP_COMPRESSOR = new NoopCompressor();
-  private static final VeniceCompressor GZIP_COMPRESSOR = new GzipCompressor();
+  private final VeniceCompressor NO_OP_COMPRESSOR = new NoopCompressor();
```

Comment:
cc: @FelixGV : Wanted to make sure that this change doesn't impact the idea behind why it was made static.

Reply:
`NO_OP_COMPRESSOR` can still stay static, right? For Gzip, it's different now that the Gzip compressor is stateful

Reply:
I think it's fine to standardize on non-static for all of them. Thanks.

Reply:
Yes, just to standardize them, thanks.